services/src/test/scala/cromwell/services/database/LiquibaseComparisonSpec.scala

```diff
@@ -0,0 +1,506 @@
+package cromwell.services.database
+
+import cromwell.core.Tags._
+import cromwell.database.slick.SlickDatabase
+import cromwell.services.database.LiquibaseComparisonSpec._
+import cromwell.services.database.LiquibaseOrdering._
+import liquibase.snapshot.DatabaseSnapshot
+import liquibase.statement.DatabaseFunction
+import liquibase.structure.DatabaseObject
+import liquibase.structure.core._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{FlatSpec, Matchers}
+import slick.jdbc.GetResult
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+import scala.reflect._
+
+/**
+  * Compares all of the various liquibase schemas against an in-memory HSQLDB-Slick schema.
```

Comment:
The point of which is to make sure they are all consistent with the schema in an RDMBS that is thoroughly unit tested?

Reply:
Sort of. It's primarily to make sure that all DBMS schemas are consistent with the HQSLDB schema.

Sure, as a secondary benefit we do get a lot of testing of HSQLDB + MySQL, but for example we still do not guarantee via tests that we can store more than 2<sup>31</sup> rows in `CALL_CACHING_HASH_ENTRY_ID`. [Slick does accept a `Long` client-side](https://github.com/broadinstitute/cromwell/blob/43/database/sql/src/main/scala/cromwell/database/slick/tables/CallCachingHashEntryComponent.scala#L12), but without the correct Liquibase that alone wasn't stopping PostgreSQL from hitting that limit server-side before this PR.